cmd/cinder-provisioner/main.go

```diff
@@ -46,6 +47,8 @@ func main() {
 	pflag.StringVar(&id, "id", "", "Unique provisioner identity")
 	pflag.StringVar(&cloudconfig, "cloud-config", "", "Path to OpenStack config file")
 
+	volumeservice.AddExtraFlags(pflag.CommandLine)
```

Comment:
Since we are adding a new flag to the API with associated functionality, can we please add unit tests to check the default and the custom behavior, including using multiple flags.

Reply:
I have added tests for the components which have the new flag; [diff](https://github.com/kubernetes/cloud-provider-openstack/compare/00ae5d268e23dd8aa5c80411a099a367d591af0c..e20a17f40ee9f19bd596f96b8a17be195e939611).